Goal: passwd: print "success" on successful password change
      (Ben Collins, 29 Apr 2000)
Fixes: #58676

Status wrt upstream: Upstream is even better, it uses gettext.

Index: shadow-4.0.3/libmisc/pam_pass.c
===================================================================
--- shadow-4.0.3.orig/libmisc/pam_pass.c	2005-05-20 00:26:52.989079000 +0200
+++ shadow-4.0.3/libmisc/pam_pass.c	2005-05-20 00:30:45.529079000 +0200
@@ -51,6 +51,7 @@
 		exit(10);  /* XXX */
 	}
 
+	fputs("passwd: password updated successfully\n", stderr);
 	pam_end(pamh, PAM_SUCCESS);
 }
 #else /* !USE_PAM */
